seed/challenges/08-coding-interview-questions-and-take-home-assignments/project-euler-problems.json

```diff
@@ -421,17 +421,129 @@
       "type": "bonfire",
       "title": "Problem 13: Large sum",
       "tests": [
-        "assert.strictEqual(euler13(), 5537376230, 'message: <code>euler13()</code> should return 5537376230.');"
+        "assert.strictEqual(largeSum(testNums), 8348422521, 'message: <code>largeSum(testNums)</code> should return 8348422521.');",
+        "assert.strictEqual(largeSum(fiftyDigitNums), 5537376230, 'message: <code>largeSum(fiftyDigitNums)</code> should return 5537376230.');"
       ],
       "solutions": [],
       "translations": {},
       "challengeSeed": [
-        "function euler13() {",
+        "function largeSum(arr) {",
         "  // Good luck!",
         "  return true;",
         "}",
         "",
-        "euler13();"
+        "// only change code above this line",
+        "",
+        "testNums = [",
+        "  '37107287533902102798797998220837590246510135740250',",
+        "  '46376937677490009712648124896970078050417018260538'",
+        "];",
+        "",
+        "fiftyDigitNums = [",
```

Comment:
The code editor is stuggling with all this text having to re-render it on every keystroke. Could we move it to the head. It makes working on the problem fustration because of the lag in the editor updating.

@BerkeleyTrue, is this symptomatic of another problem?

Reply:
@Bouncey, thank you very much for your review and suggestions. I did notice the lag when testing it out on my machine--didn't realize it was because the page was re-rendering like that!

Quick question about moving the large number array to the head, can I do that from this seed file? Would it involve moving the array to the test portion of the problem? Above the assert statements?

Also, I'll check my other PRs which may cause similar lag with all the re-rendering.

Reply:
Yes from the see file. `head` is a property on the challenge object that is an array of strings, much like the `challengeseed` property. It is used to set up a challenge, out of scope of the user.

[head example](https://github.com/freeCodeCamp/freeCodeCamp/blob/4346de64f3b82e447030d1108f8213ec5b1ee422/seed/challenges/08-coding-interview-questions-and-take-home-assignments/coding-interview-data-structure-questions.json#L911)

Reply:
Also, just I've only just noticed. We will need these arrays assigned correctly.
```js
const testNums = [ ... ];
const fiftyDigitNums = [ ... ];
```

Reply:
Okay! Thank you for your explanation and example of the head property. I'll get on it and will update this PR soon.